content/guides/applications/web-app-integrations/configure.md

```diff
@@ -84,18 +74,38 @@ The following parameters are available.
-| `file_extension`      | The extension of the file.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| `auth_code`           | The OAuth 2.0 [authorization code][code], which is generated by Box upon successful authentication. Your application must then supply this authorization code to Box in exchange for an OAuth 2.0 Access Token. An authorization header containing a valid Access Token must be included in every Box API request.                                                                                                                                                                                                                                                       |
+| `user_id` | The Box [user ID][uid]. This information is used in Popup Integrations in which user authentication is required to complete an action. You can store the Box ID in your application to enable subsequent authentication requests from the integration.|
+| `user_name` | The full name or email address of the Box user. Not all Box users specify their names at all times. |
+| `file_id` | The Box [file ID][fid]. You can use this ID to make Box API calls that affect the file. |
+| `file_name` | The name of the file.  |
+| `file_extension` | The extension of the file. |
+| `auth_code` | The OAuth 2.0 [authorization code][code], which is generated by Box upon successful authentication. Your application must then supply this authorization code to Box in exchange for an OAuth 2.0 Access Token. An authorization header containing a valid Access Token must be included in every Box API request. 
 | `redirect_to_box_url` | In Popup Integrations, the URL to which requests are sent by the confirmation prompt. Use this URL to redirect users to the All Files page. This parameter closes the popup panel and refreshes the All Files page to reflect any changes performed by the integration. If you do not want to add this parameter to your application, you can specify the entire URL. **Success**: `#redirect_to_box_url#&status=success&message=Your%20action%20was%20successful%2E`. **Failure**: `#redirect_to_box_url#&status=failure&message=Your%20action%20was%20unsuccessful%2E` |
```

Comment:
There is a formatting issue at the end of this paragraph (see in preview mode). `Success` and `Failure` examples should start from a new line.